src/main/java/core/basesyntax/service/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service;
+
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Color getColor() {
+        Color[] colors = Color.values();
```

Comment:
This variable redundant, just use Color.values().[index] when need to get some color by index